src/components/common/fields/InputField.jsx

```diff
@@ -66,7 +68,7 @@ function InputField(props) {
     <Col>
       <Row width="auto" margin="0" >
         <label style={{ ...defaultTitleStyle, ...titleStyle }}>
-          {title}{required ? '*' : ''}
+          {label ? title : ''}{required && label ? '*' : ''}
```

Comment:
Always prefer `null `over empty string wherever possible.
```suggestion
          {label ? title : null}{required && label ? '*' : null}
```

Reply:
Done.